server/lib/packets/rescue.go

```diff
@@ -3,23 +3,23 @@ package packets
 import (
 	"log"
 
-	proto "../../proto"
-	"../models"
+	proto "github.com/umegaya/channer/server/proto"
+	"github.com/umegaya/channer/server/lib/models"
 )
 
 func ProcessRescue(from Source, msgid uint32, req *proto.RescueRequest, t Transport) {
 	a, err := models.FindAccount(models.DBM(), req.Account)
 	if err != nil {
-		SendError(from, msgid, proto.Error_Rescue_DatabaseError)
+		SendError(from, msgid, &proto.Err{ Type: proto.Error_Rescue_DatabaseError })
 		return
 	}
 	if !a.VerifySign(req.Sign, req.Walltime) {
-		SendError(from, msgid, proto.Error_Rescue_InvalidAuth)
+		SendError(from, msgid, &proto.Err{ Type: proto.Error_Rescue_InvalidAuth })
```

Comment:
__[gometalinter]__
* [gotype] undeclared name: proto
* [gotype] cannot pass argument &(proto.Err literal) (value of type *invalid type) to parameter of type error